Sources/PackageGraph/PackageGraphLoader.swift

```diff
@@ -273,10 +273,13 @@ private func createResolvedPackages(
             // If a target with similar name was encountered before, we emit a diagnostic.
             let targetName = targetBuilder.target.name
             if allTargetNames.contains(targetName) {
-                diagnostics.emit(ModuleError.duplicateModule(targetName), location: diagnosticLocation())
+                let targets = packageBuilders
+                    .filter({ $0.targets
+                        .contains { $0.target.name == targetName }})
+                    .map({ $0.package.name })
```

Comment:
How about formatting to this:

```swift
// Find the packages this target is present in.
let packages = packageBuilders
    .filter({ $0.targets.contains(where: { $0.target.name == targetName }) })
    .map({ $0.package.name })
```

Reply:
Yikes, good save! We are getting the package name, so renaming the variable is 👏